framework/db/src/com/cloud/utils/db/TransactionLegacy.java

```diff
@@ -1014,6 +1014,13 @@ public static void initDataSource(Properties dbProps) {
             if (dbProps.size() == 0)
                 return;
 
+            try {
+                // Explicitely load JDBC driver because it has been removed from the classpath as part of commit c22659d76d73f00f41c13776c490e17a50aacd20
+                Class.forName("com.mysql.jdbc.Driver");
```

Comment:
If you can make it work without noticing the time you spend on it, please. I would go for db.driver.path btw, so we don't have to maintain a driver table in the code.